dashboard/db/migrate/20170626160941_add_storage_app_to_channel_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddStorageAppToChannelTokens < ActiveRecord::Migration[5.0]
+  def change
+    add_column :channel_tokens, :storage_app_id, :integer, after: :channel
```

Comment:
This is a good point. There is currently no plan to rename storage_apps, so let's keep it as-is.